pyproject.toml

```diff
@@ -33,4 +33,4 @@ environment = { CMAKE_GENERATOR = "Visual Studio 16 2019"}
 # For older Pythons we maintain our manylinux2010 support.
 [[tool.cibuildwheel.overrides]]
 select = "cp3{6,7,8,9}-manylinux*"
-manylinux-x86_64-image = "manylinux2010"
+# manylinux-x86_64-image = "manylinux2010"
```

Comment:
I fixed this for real in #1498 (and a corresponding fix for the stable branch in #1499)
```suggestion
```